ci/travis/build-docker-images.py

```diff
@@ -0,0 +1,230 @@
+import datetime
+import glob
+import os
+import re
+import shutil
+import subprocess
+import sys
+from typing import List
+
+import docker
+
+DOCKER_USERNAME = "raytravisbot"
+DOCKER_CLIENT = None
+
+
+def _subprocess_wrapper(cmd):
+    result = subprocess.run(
```

Comment:
seems like we only use this to run python. how about [runpy](https://docs.python.org/3/library/runpy.html#runpy.run_path)